src/styles/dotcms-theme/components/_calendar.scss

```diff
@@ -1,261 +1,310 @@
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
 }
+
 .p-datepicker table td.p-datepicker-today > span {
-  background: $white;
-  color: $black;
-  border-color: $gray-light;
+    background: $white;
+    color: $black;
+    border-color: $gray-light;
 }
+
 .p-datepicker table td.p-datepicker-today > span.p-highlight {
-  color:  $highlight-text-color;
-  background: $highlight-bg;
+    color: $highlight-text-color;
+    background: $highlight-bg;
 }
+
 .p-datepicker .p-datepicker-buttonbar {
-  padding: $spacing-2 0;
-  border-top: 1px solid $bg-color-secondary;
+    padding: $spacing-2 0;
+    border-top: 1px solid $bg-color-secondary;
 }
+
 .p-datepicker .p-datepicker-buttonbar .p-button {
-  width: auto;
+    width: auto;
 }
+
 .p-datepicker .p-timepicker {
-  border-top: 1px solid $bg-color-secondary;
-  padding: $spacing-1;
+    border-top: 1px solid $bg-color-secondary;
+    padding: $spacing-1;
 }
+
 .p-datepicker .p-timepicker button {
-  width: $spacing-5;
-  height: $spacing-5;
-  color: rgba(0, 0, 0, 0.6);
-  border: 0 none;
-  background: transparent;
-  border-radius: 50%;
-  transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
+    width: $spacing-5;
+    height: $spacing-5;
+    color: rgba(0, 0, 0, 0.6);
+    border: 0 none;
+    background: transparent;
+    border-radius: 50%;
+    transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
 }
+
 .p-datepicker .p-timepicker button:enabled:hover {
-  color: rgba(0, 0, 0, 0.6);
-  border-color: transparent;
-  background: $bg-color-secondary;
+    color: $hover-text-color;
+    border-color: transparent;
+    background: $hover-bg;
 }
+
 .p-datepicker .p-timepicker button:focus {
-  outline: 0 none;
-  outline-offset: 0;
-  box-shadow: none;
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
 }
+
 .p-datepicker .p-timepicker button:last-child {
-  margin-top: 0.2em;
+    margin-top: 0.2em;
 }
+
 .p-datepicker .p-timepicker span {
-  font-size: $spacing-4;
+    font-size: $spacing-4;
 }
+
 .p-datepicker .p-timepicker > div {
-  padding: 0 $spacing-1;
+    padding: 0 $spacing-1;
 }
+
 .p-datepicker.p-datepicker-timeonly .p-timepicker {
-  border-top: 0 none;
+    border-top: 0 none;
 }
+
 .p-datepicker .p-monthpicker {
-  margin: $spacing-1 0;
+    margin: $spacing-1 0;
 }
+
 .p-datepicker .p-monthpicker .p-monthpicker-month {
-  padding: $spacing-1;
-  transition: none;
-  border-radius: 4px;
+    padding: $spacing-1;
+    transition: none;
+    border-radius: 4px;
 }
+
 .p-datepicker .p-monthpicker .p-monthpicker-month.p-highlight {
-  color:  $highlight-text-color;
-  background: $highlight-bg;
+    color: $highlight-text-color;
+    background: $highlight-bg;
 }
+
 .p-datepicker.p-datepicker-multiple-month .p-datepicker-group {
-  border-right: 1px solid $bg-color-secondary;
-  padding-right: $spacing-1;
-  padding-left: $spacing-1;
-  padding-top: 0;
-  padding-bottom: 0;
+    border-right: 1px solid $hover-bg;
```

Comment:
bg-color in a border. maybe is because we want the border to be the same as the background of the element? if that;s the case can we use transparent?